ocaml/testsuite/tests/typing-local/regions.ml

```diff
@@ -18,6 +18,37 @@ let check_empty name =
 
 let () = check_empty "startup"
 
+let foo x = local_
```

Comment:
Slightly better to use an informative name here. How about `allocate_in_parent_region` for this one and `leak` for `foo_bad`?